pceplib/pcep_msg_objects.h

```diff
@@ -390,6 +390,8 @@ struct pcep_object_vendor_info {
 	struct pcep_object_header header;
 	uint32_t enterprise_number;
 	uint32_t enterprise_specific_info;
+	/* cisco sends color for PcInit */
+	uint32_t enterprise_color;
```

Comment:
Vendor info objects are not working like that. You must keep vendor object generic and in function of the entreprise number decides to parse the rest of the PCEP Vendor Info Object to match the Cisco, Juniper, Nokia ... specific values

Reply:
I'll add to the vendor object generic fields.